content/channels/index.textile

```diff
@@ -479,6 +479,16 @@ else:
     print('An error occurred; err =', response.error_message)
 ```
 
+```[rest_php]
+$rest = new Ably\AblyRest(
+    ['key' => '{{API_KEY}}']
+);
+$content = ['channels' => ['test1', 'test2'], 'messages' => ['data' => 'myData']];
+$batchPublish = $rest->request('POST', '/messages', [], $content);
+
+echo('Success! status code was ' . $batchPublish->statusCode);
```

Comment:
Link to the batch publish doc can be added at the end => https://github.com/ably/ably-php/?tab=readme-ov-file#making-explicit-http-requests-to-ably-rest-endpoints--batch-publish

Reply:
I'm not sure this is necessary, it's the same information, usually we try to avoid linking away from the docs unless unavoidable.

Reply:
Okay, sgtm